src/app.rb

```diff
@@ -0,0 +1,125 @@
+require './book'
+require './classroom'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def all_booklist
+    puts 'There is no book found in the store' if
+    @books.empty?
+    @books.each { |book| puts "Title: #{book.title}, Author: #{book.author}" }
+  end
+
+  def all_peoplelist
+    puts 'No people found!' if @people.empty?
+    @people.each { |person| puts "[#{person.class}], Age: #{person.age}, Name: #{person.name}, Id: #{person.id}" }
+  end
+
+  def create_person
+    print 'Missing student or teacher? create student (1) or a teacher (2)
+        [Input the number]: '
+    option = gets.chomp
+    case option
+    when '1'
+      create_student
+    when '2'
+      create_teacher
+    else
+      puts 'invalid input'
+    end
+    puts 'Person created successfully'
+  end
+
+  def create_teacher
+    print 'teacher age: '
+    age = gets.chomp
+
+    print 'teacher specialization: '
+    specialization = gets.chomp
+
+    print 'teacher name: '
+    name = gets.chomp
+
+    teacher = Teacher.new(age, specialization, name, parent_permission: true)
+    puts 'New teacher was created succefully'
+    @people.push(teacher)
+  end
+
+  def create_student
+    print 'student age: '
+    age = gets.chomp
+
+    print 'student name: '
+    name = gets.chomp
+
+    print 'Parent permission [Yes or No]: '
+    parent_permission = gets.chomp.downcase
+
+    case parent_permission
+    when 'no'
+      student = Student.new(age, name, parent_permission: false)
+      @people.push(student)
+    when 'yes'
+      student = Student.new(age, name, parent_permission: true)
+      @people.push(student)
```

Comment:
- [ ] Here you are creating a Student without a classroom attribute, so the name declared on the command line is being saved as a classroom. You can resolve this by requesting a classroom from the user:

  ```
  print 'student classroom: '
  classroom = gets.chomp
  ```
  Next your modify this objects to take in the classroom.
  ```
  ...
  when 'no'
    student = Student.new(age, classroom, name, parent_permission: false)
    @people.push(student)
  when 'yes'
    student = Student.new(age, classroom, name, parent_permission: true)
  ...
  ```

  ![kfo](https://user-images.githubusercontent.com/54848363/195630478-2fea66bf-e4f9-4af0-a76c-60c6f6db3221.jpg)

Reply:
> * [x]  Here you are creating a Student without a classroom attribute, so the name declared on the command line is being saved as a classroom. You can resolve this by requesting a classroom from the user:
>   ```
>   print 'student classroom: '
>   classroom = gets.chomp
>   ```
>   
>   
>       
>         
>       
>   
>         
>       
>   
>       
>     
>   Next you modify these objects to take in the classroom.
>   ```
>   ...
>   when 'no'
>     student = Student.new(age, classroom, name, parent_permission: false)
>     @people.push(student)
>   when 'yes'
>     student = Student.new(age, classroom, name, parent_permission: true)
>   ...
>   ```
>   
>   
>       
>         
>       
>   
>         
>       
>   
>       
>     
>   ![kfo](https://user-images.githubusercontent.com/54848363/195630478-2fea66bf-e4f9-4af0-a76c-60c6f6db3221.jpg)

- [ ] You haven't implemented this suggestion by the previous reviewer. Kindly work on it.